deps/oob-1.0.2/jsonutil.c

```diff
@@ -16,27 +17,28 @@
 #define TMP_BUF_LEN 40
 #endif
 
-//static struct conStr connectionInfo ;
 static struct conStr connectionInfo = {{0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0}, {0} };
 
 static struct logDetails oobevent = {{0}, {0}, {0}, {0}, 0, 0};
 
-void setdeployment(const char *host);
+cJSON* dsn = NULL;
```

Comment:
You need free the data this pointer refers to after the event has been sent. Check out https://github.com/snowflakedb/libsnowflakeclient/blob/994a2b2c63837c8c41eff3113100602d3ee656f5/deps/oob-1.0.2/jsonutil.c#L487